mujoco_warp/_src/forward.py

```diff
@@ -142,6 +142,34 @@ def _next_velocity(
   qvel_out[worldid, dofid] = qvel_in[worldid, dofid] + qacc_scale_in * qacc_in[worldid, dofid] * timestep
 
 
+@wp.func
+def _next_act(
+  # Model:
+  opt_timestep: float,  # kernel_analyzer: ignore
```

Comment:
let's add a TODO to decide whether kernel analyzer should accept slices of arrays and if so, should we have some kind of naming requirement to indicate that it's a slice of an array

Reply:
added todos in contrib/kernel_analyzer/kernel_analyzer/ast_analyzer.py and forward.py for array slice